alf/utils/summary_utils.py

```diff
@@ -165,6 +165,39 @@ def add_nested_summaries(prefix, data):
             alf.summary.scalar(name, elem)
 
 
+@_summary_wrapper
+def summarize_per_category_loss(loss_info: LossInfo):
```

Comment:
perhaps can add an optional argument: ``label_name: Optional[List[str]]=None`` to use label name for the summary name. ``label_name`` can be configured using ``alf.config()``

Reply:
Added label_label arguments and now the summarization could be more informative, as shown in the PR description.